java-lib/src/test/java/com/wavefront/data/ValidationTest.java

```diff
@@ -441,18 +441,13 @@ public void testInvalidSpansWithValidationConfig() {
       assertTrue(iae.getMessage().contains("WF-432"));
     }
 
-    // span annotation value too long: WF-433
+    // accept long span annotation value
     span = getValidSpan();
     span.getAnnotations().add(new Annotation("k", "v23456789012345"));
     Validation.validateSpan(span, config);
     span = getValidSpan();
     span.getAnnotations().add(new Annotation("k", "v234567890123456"));
-    try {
```

Comment:
apart from deleting current test case, please add new test case for the code that you have changed.